gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -199,6 +220,10 @@
 
     log.info("Finding copy entities for table " + dataset.table.getCompleteName());
 
+    this.eventSubmitter = new EventSubmitter.Builder(dataset.getMetricContext(), "hive.dataset.copy").build();
+    Closer closer = Closer.create();
```

Comment:
closer init should be in try block and closer.closer() in finally to be on the safer side